internal/cloudapi/v2/server.go

```diff
@@ -458,7 +458,7 @@ func generateManifest(ctx context.Context, workers *worker.Server, depsolveJobID
 		options.OSTree.URL = result.Specs[0].URL
 	}
 
-	manifest, err := imageType.Manifest(b, options, repos, depsolveResults.PackageSpecs, containerSpecs, seed)
+	manifest, _, err := imageType.Manifest(b, options, repos, depsolveResults.PackageSpecs, containerSpecs, seed)
```

Comment:
I just realized if it wouldn't make sense to extend the `JobResult` with a `Warnings` property and return any warnings back.

Specifically in https://github.com/osbuild/osbuild-composer/blob/d7e2e5d7400dad8a88553c6d23582eaa84ce2b27/internal/worker/json.go#L42-L44

Probably a question for @croissanne if we want to eventually return warnings also from the Cloud API and image builder? 🤔 It probably does not make that much sense, since we have more control over the allowed customizations exposed in image-builder, but it may make sense for Koji integration.

Reply:
FWIW I think it makes sense.

Reply:
I think we can always add it. I'm all for expanding the JobResult base struct to be a bit more verbose :)